### ISSUE TITLE
addStatements fails if containing block is only indented a portion of an indentation level

### ISSUE THREAD

**Issue by cancerberoSgx** [dsherret/ts-simple-ast#335]
I'm kind of short of time to investigate this further - so I just report it so I don't loose track of this: If you try to addStatement() to a target method/function/arrow that is already indented it will break: 

```ts 

import Project from 'ts-simple-ast'
  const project = new Project()
  const sourceFile = project.createSourceFile('created.ts', `
function nonIndented(){
}
  function indentedF(){
  
  }
`)
sourceFile.getFunction('nonIndented').addStatements('return null;')
console.log(sourceFile.getText())


sourceFile.getFunction('indentedF').addStatements('return null;')
console.log(sourceFile.getText())

```
calling addStatements on nonIndented function works nut on indentedF fails throwing this exceptino:

```sh

/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:380
        this._text += Array(this._currentIndentation + 1).join(this._indentationText);
                      ^
RangeError: Invalid array length
    at CodeBlockWriter._writeIndentation (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:380:23)
    at CodeBlockWriter.writeIndividual (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:317:26)
    at /home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:311:29
    at Array.forEach (<anonymous>)
    at CodeBlockWriter._writeIndentingNewLines (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:306:15)
    at CodeBlockWriter.write (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/code-block-writer/dist/code-block-writer.js:216:14)
    at printTextFromStringOrWriter (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/ts-simple-ast/dist/utils/getTextFromStringOrWriter.js:11:16)
    at Object.getTextFromStringOrWriter (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/ts-simple-ast/dist/utils/getTextFromStringOrWriter.js:5:5)
    at SyntaxList.insertChildText (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/ts-simple-ast/dist/compiler/common/SyntaxList.js:22:34)
    at FunctionDeclaration.StatementedNode.class_1.insertStatements (/home/sg/git/typescript-plugins-of-mine/typescript-plugin-proactive-code-fixes/node_modules/ts-simple-ast/dist/compiler/statement/StatementedNode.js:37:53)

```

**Comment by dsherret** [comment]
Thanks! I took a look and it's because the result of getting the indentation is 1.5. I didn't know it could be that. I'll fix code-block-writer to handle this situation.

**Comment by dsherret** [comment]
Fixed in 11.2.1.
